src/olympia/activity/models.py

```diff
@@ -162,6 +162,20 @@ class Meta:
         ordering = ('-created',)
 
 
+class DraftComment(ModelBase):
+    """A model that allows us to draft comments for reviews before we have
+    an ActivityLog instance ready.
+
+    This is being used by the commenting API by the code-manager.
+    """
+    id = PositiveAutoField(primary_key=True)
+    comments = models.TextField()
```

Comment:
Sorry for the naive question: is bleach used somewhere so that we do not store XSS in the db?

Reply:
I checked most of the templates that render the comments and they're using `|safe`, except for `reviewers/includes/history.html` which is rendering activity log entries, indeed. Let me play around a bit with XSS and check if we should sanitize this, we probably should but given that the developer may quote code-parts and we're rendering it to HTML and email this may be tricky.